mitiq/pec/tests/test_pec.py

```diff
@@ -110,7 +110,7 @@ def test_execute_with_pec_with_different_samples(circuit: Circuit, seed):
     assert np.average(errors_more_samples) < np.average(errors_few_samples)
 
 
-@mark.parametrize("num_samples", [100, 1000])
+@mark.parametrize("num_samples", [100, 1000, 100001])
```

Comment:
Instead of changing an existing test, I would add a new test dedicated to the new warning.
There is a specific way to test that warnings, for example you can have a look at this similar test: https://github.com/unitaryfund/mitiq/blob/e96e8a1ce50a9a70f215cd35c3ce24957a60e0d7/mitiq/zne/tests/test_inference.py#L605